src/props/getTokenList.ts

```
@@ -52,7 +52,8 @@ const fixTotkens = (tokenlist) => {
 const markMultichain = async (tokens) => {
 	const multichainList = await fetch(`https://bridgeapi.multichain.org/v4/tokenlistv4/${FANTOM_ID}`)
 		.then((r) => r.json())
-		.then((r) => Object.values(r));
+		.then((r) => Object.values(r))
+		.catch(() => []);
```

this is important enough that if it fails it would be better to fail the whole deployment rather than return an empty list